common-header.tex

```diff
@@ -190,8 +195,8 @@
 
 \TOCLevels{chapter}{section}
 \begin{minitocfmt}{\chapmtoc}
-\declaretocfmt{section}{\@F{\color{white}\hspace{1.0cm}\textbullet\hspace{0.25cm}\Large\bfseries}\@B{2.8cm}\@P{\mtocgobble}}
-\declaretocfmt{section*}{\@F{\color{white}\hspace{1.0cm}\textbullet\hspace{0.25cm}\Large\bfseries}\@B{2.8cm}\@P{\mtocgobble}}
+\declaretocfmt{section}{\@F{\color{white}\hypersetup{linkcolor=white}\hspace{1.0cm}\textbullet\hspace{0.25cm}\Large\bfseries}\@B{2.8cm}\@P{\mtocgobble}}
+\declaretocfmt{section*}{\@F{\color{white}\hypersetup{linkcolor=white}\hspace{1.0cm}\textbullet\hspace{0.25cm}\Large\bfseries}\@B{2.8cm}\@P{\mtocgobble}}

```

Comment:
Here and in previous line, one can replace `linkcolor=white` by `hidelinks`. This was my initial idea but it took time until I understood how the Chapter TOCs were handled and I initially used `\hypersetup{hidelinks}` at a wrong place.  When I found the correct place (here), I inserted `linkcolor=white` rather. Both work.